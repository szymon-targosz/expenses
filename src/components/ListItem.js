import React from 'react';

const ListItem = ({ description, note, amount, createdAt}) => (
    <div>
        <h3>{description}</h3>
        <p>{amount} - {createdAt}</p>
    </div>
);

export default ListItem;