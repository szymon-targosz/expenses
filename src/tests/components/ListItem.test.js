import React from 'react';
import { shallow } from 'enzyme';
import ListItem from '../../components/ListItem';
import expenses from '../fixtures/expenses';

test('should render ListItem correctly', () => {
    const wrapper = shallow(<ListItem {...expenses[1]}/>);
    expect(wrapper).toMatchSnapshot();
});