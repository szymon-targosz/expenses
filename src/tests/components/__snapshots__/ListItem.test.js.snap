// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ListItem correctly 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      Gas bill
    </h3>
    <span
      className="list-item__subtitle"
    >
      December 30th, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $35.90
  </h3>
</Link>
`;
